VulkanLaunchpad.h

```diff
@@ -204,11 +209,21 @@ extern const char *vklRequiredInstanceExtensions[];
 const char **vklGetRequiredInstanceExtensions(uint32_t *out_count);
 
 /*!
- * Initializes the framework
+ *  Initializes the framework
  */
 bool vklInitFramework(VkInstance vk_instance, VkSurfaceKHR vk_surface, VkPhysicalDevice vk_physical_device,
                       VkDevice vk_device, VkQueue vk_queue, const VklSwapchainConfig &swapchain_config);
 
+#ifdef VKL_HAS_VMA
+/*!
+ *  Initializes the framework.
+ *  This overload additionally takes a VmaAllocator handle
+ */
+bool vklInitFramework(VkInstance vk_instance, VkSurfaceKHR vk_surface, VkPhysicalDevice vk_physical_device,
```

Comment:
Should the user really create the vma allocator? We can derive everything from the given inputs that is needed

Reply:
Yeah, we could... except for creating a depth buffer image, which must happen before the call to `vklInitFramework`. 🤷‍♂️ 

Therefore, I thought that it might be the best approach to:
- Let Vulkan Launchpad support VMA if it _can_ be found by the compiler.
- Make it opt-in
- The user can always use it to create resources manually, even before the call to `vklInitFramework`---otherwise, how to let the user have VMA allocate memory for the depth buffer image?

Also, letting the user setup VMA manually coulld be argues to serve a didactic purpose.

Reply:
I though vma should happen in the background and will not be noticeable by the user. So the user interaction than is kinda 50/50 with vma?

Reply:
Yeah, my first approach was 99% trsnsparent (besides a flag/switch to enable it).

But then I switched to 50/50 for the reasons stated above. What do you think?

Reply:
And you know what's cool?!
If you don't do anything, everything stays the same as before.
But if you create a VMA allocator, the you can use VMA---I think, this would make a great addition to the depth buffer task.